pkg/controllers/hub/trafficmanagerbackend/controller.go

```diff
@@ -89,6 +92,22 @@ var (
 		Name:      "traffic_manager_backend_status_last_timestamp_seconds",
 		Help:      "Last update timestamp of traffic manager backend status in seconds",
 	}, []string{"namespace", "name", "generation", "condition", "status", "reason"})
+
+	// trafficManagerARMAPILatency is a prometheus metric that measures the latency of ARM API calls in milliseconds.
+	trafficManagerARMAPILatency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
reuse the https://github.com/kubernetes-sigs/cloud-provider-azure/blame/master/pkg/azclient/metrics/metrics.go this package and do similar to what fileshareclient does, https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/pkg/azclient/fileshareclient/custom.go